jiwonjang/app.js

```diff
@@ -0,0 +1,40 @@
+const dotenv = require("dotenv");
+dotenv.config();
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  res.json({ message: "pong" });
+});
+
+const server = http.createServer(app);
+const PORT = process.env.PORT;
+
+const start = async () => {
+  server.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```

Comment:
```suggestion
  app.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```
http로 만들어준 server 변수 대신에 express로 만들어준 app 변수를 사용하면 됩니다!